scripts/s_aureus/collate_results.py

```diff
@@ -49,6 +49,15 @@ def collate_results():
             # All of these channel results csv files are single sample
             output_results_csv_list.append(next(csv.DictReader(channel_csv_file)))
 
+    # Add on any failed samples where the container failed completely and did not produce a results.csv for that sample.
+    # A sample is failed if there are no results in output_results_csv_list, but sample is in the input metadata file
+    samples_with_output = set(r["SAMPLE_ID"] for r in output_results_csv_list)
+    with open("metadata.csv") as input_metadata_file:
+        for md_sample in csv.DictReader(input_metadata_file):
+            if md_sample["SAMPLE_ID"] not in samples_with_output:
+                print("Failed sample: " + str(md_sample))
```

Comment:
any reason your using print and not a logger here?

Reply:
Hangover from "coding fast" for the last 15 days.
The logger and logfile are not set up in this script.
Easiest fix is to not output failed samples to the log. I'll do that.

Reply:
"fixed"